src/api/app/models/kiwi/preference_type.rb

```diff
@@ -0,0 +1,38 @@
+class Kiwi::PreferenceType < ApplicationRecord
+  belongs_to :image, inverse_of: :preference_type
+
+  enum image_xml: %i[btrfs clicfs cpio docker ext2 ext3 ext4 iso lxc oem product pxe reiserfs split squashfs tbz vmx xfs zfs]
```

Comment:
@evanrolfe, Is valid to have a "containerconfig" with any of this "image_xml" type?

Reply:
Its valid in our models yes, as for Kiwi I'm not sure if its valid or not since containerconfig isn't mentioned anywhere in the [scema docs](https://doc.opensuse.org/projects/kiwi/schema-doc/) at all.. Maybe @adrianschroeter knows?

Reply:
@adrianschroeter, its valid?

Reply:
No, containerconig is only valid in combination with ``docker`` (and maybe also lxc)

Reply:
We should also reflect that in the UI to only show it for container builds.
To be honest I don't understand why we now start to implement this. I would prefer to finish the image story (sure, docker container are also just kiwi images but I think we're mixing things up here...)